core/templates/components/entity-creation-services/exploration-creation.service.ts

```diff
@@ -39,7 +39,7 @@ angular.module('oppia').factory('ExplorationCreationService', [
     var explorationCreationInProgress = false;
 
     return {
-      createNewExploration: function() {
+      createNewExploration: function(params = {}) {
```

Comment:
Also, can you please add a unit test for creating a new exploration with this function with title param?

Reply:
sir there is no spec.ts file for this file.